frontend/src/metabase/actions/components/ActionViz/ExplainerText/tests/premium.unit.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { screen } from "__support__/ui";
+import { setup as baseSetup } from "./setup";
+import type { SetupOpts } from "./setup";
+
+function setup(opts: SetupOpts) {
+  baseSetup({
+    hasEnterprisePlugins: true,
+    tokenFeatures: { whitelabel: true },
+    ...opts,
+  });
+}
+
+describe("ExplainerText (EE without token)", () => {
+  it("should render help link when `show-metabase-links: true`", () => {
+    setup({ showMetabaseLinks: true });
+
+    expect(
+      screen.getByText(
+        "You can either ask users to enter values, or use the value of a dashboard filter.",
+      ),
+    ).toBeInTheDocument();
+    expect(screen.getByText("Learn more.")).toBeInTheDocument();
+  });
+
+  it("should not render help link when `show-metabase-links: false`", () => {
```

Comment:
Also `premium.unit.spec.ts` are usually the only file that asserts that the link won't be rendered. There are some exceptions like map visualization [(explained here)](https://metaboat.slack.com/archives/C063Q3F1HPF/p1705052068310819?thread_ts=1704984986.574009&cid=C063Q3F1HPF).